src/main/resources/default/assets/tycho/scripts/selectableTable.js.pasta

```diff
@@ -0,0 +1,49 @@
+sirius.ready(function () {
+    document.querySelectorAll('.select-all-visible-table-rows-checkbox-js').forEach(function (_selectAllCheckbox) {
+        const _table = _selectAllCheckbox.closest('table');
+        // Handle changing the state of the select-all checkbox found in the table head row
+        _selectAllCheckbox.addEventListener('change', function () {
+            _table.querySelectorAll('.select-table-row-checkbox-js').forEach(_checkbox => {
+                _checkbox.checked = _selectAllCheckbox.checked;
+                toggleRowSelection(_checkbox);
+            });
+        });
+
+        _table.querySelectorAll('.select-table-row-checkbox-js').forEach(function (_checkbox) {
+            // Handle changing the state of the select-all checkbox found in the table head row
+            // When all rows are selected, the select-all checkbox should be checked as well
+            _checkbox.addEventListener('change', function () {
+                toggleRowSelection(_checkbox);
+                changeSelectAllCheckboxState(_table);
+            });
+
+            const _tr = _checkbox.closest('tr');
+            // Handle clicks on the row for convenience
+            _tr.addEventListener('click', function (event) {
+                if (event.target.tagName.toLowerCase() !== 'input' &&
+                    event.target.tagName.toLowerCase() !== 'a' &&
+                    event.target.tagName.toLowerCase() !== 'button') {
+                    _checkbox.checked = !_checkbox.checked;
+                    _checkbox.dispatchEvent(new Event('change'));
+                }
+            });
+
+        });
+    });
+
+    function toggleRowSelection(_checkbox) {
+        const _tr = _checkbox.closest('tr');
+        if (_checkbox.checked) {
+            _tr.style.backgroundColor = 'lightgray';
+        } else {
+            _tr.style.backgroundColor = '';
```

Comment:
**Question:** Why not via a color marker class? Also the used color is not really a "sirius" color.